core/src/com/unciv/utils/Debug.kt

```diff
@@ -5,9 +5,12 @@ object DebugUtils {
     /**
      * This exists so that when debugging we can see the entire map.
      * Remember to turn this to false before commit and upload!
+     * Or use the "secret" debug page of the options popup instead.
      */
     var VISIBLE_MAP: Boolean = false
 
+    var PAINT_COORDS: Boolean = false
```

Comment:
Instead of "paint" how about "show coords on map"? This name is unnecessarily confusing